### PR TITLE
added social media buttons to event pages

### PR DESCRIPTION
**Pull request by Iepoev**

### CODE REVIEW

File: config/locales/social_share_button.zh-CN.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'zh-CN':
```
Comment (Silox) [inline config/locales/social_share_button.zh-CN.yml]: Do we really need these translations for CN?

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -33,6 +33,10 @@
         <th>Contact e-mail</th>
         <td><%= mail_to @event.contact_email, @event.contact_email %></td>
       </tr>
+      <tr>
+        <th>Share this event</th>
+        <td><%= render_shareable :buttons=> ['twitter', 'facebook', 'google_plus'] %></td>
```
Comment (Silox) [inline app/views/events/show.html.erb]: Use corret ruby syntax here: `buttons: %w(twitter facebook google_plus)`

**Comment by NoctuaNivalis** [comment]
What happens if one clicks these buttons for events without social integration?

**Comment by mcbloch** [comment]
We'd suggest removing the deprecated google plus button.

**Comment by Silox** [comment]
Closed as stale, reopen if wanted.

**Pull request closed without merging**